js/modules/variwide.src.js

```diff
@@ -264,7 +264,7 @@ addEvent(H.Axis, 'afterDrawCrosshair', function (e) {
 addEvent(H.Axis, 'afterRender', function () {
     var axis = this;
 
-    if (!this.horiz && this.variwide) {
+    if (!this.horiz && this.variwide && this.options.labels.enabled) {
```

Comment:
Instead of checking the `enabled` option, I think it would be safer to just filter the `tickPositions` by the ones that have a label, before we do the `map`. We do something similar in RadialAxis: https://github.com/highcharts/highcharts/blob/v7.2.0/ts%2Fparts-more%2FRadialAxis.ts/#L700-L704 .